### PR TITLE
Allow the no_course_image to come from the current theme

### LINKED ISSUE #67: No Course Image Hardcoded for Indigo Theme
The no_course_image.png is hardcoded for Indigo instead of reference the theme in use

### PR DESCRIPTION
**Pull request by misilot**
Fixes #67

### CODE REVIEW

File: tutorindigo/templates/indigo/lms/templates/discovery/course_card.underscore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <article class="course" role="region" aria-label="<%- content.display_name %>">
         <header class="course-image">
             <div class="cover-image">
-                <img src="<%- image_url %>" onerror="this.src='/static/indigo/images/no_course_image.png';" alt="<%- content.display_name %> <%- content.number %>" />
+                <img src="<%- image_url %>" onerror="this.src='${static.url('images/no_course_image.png')};" alt="<%- content.display_name %> <%- content.number %>" />
```
Comment (hinakhadim) [inline tutorindigo/templates/indigo/lms/templates/discovery/course_card.underscore:4]: Hi, @misilot. This is not working fine as the file's extension is `underscore` and its a `mako` template. It doesn't support `static.url` that's why I have to hardcode the path. The above change in `course.html` is fine as its a `html` template. Can you please test it on your system?

Comment (misilot) [reply tutorindigo/templates/indigo/lms/templates/discovery/course_card.underscore:4]: Sure, can you give me an example of where I can find this in the LMS? As the two places I thought they might be, it _was_ working.

I can update it to `/theming/asset/images/no_course_image.png` if that would make more sense, so it isn't hardcoded to the specific theme?

Comment (hinakhadim) [reply tutorindigo/templates/indigo/lms/templates/discovery/course_card.underscore:4]: I also haven't an example of this in LMS. In the first place, it's perfectly working. In second case `course_card.underscore`, it's redering static.url as string.

We can update to `/theming/asset/images/no_course_image.png` (as it's working fine), but the problem is `no_course_image.png` currently exists in only indigo theme. Lets update to `/theming/asset/images/no_course_image.png`.

Comment (misilot) [reply tutorindigo/templates/indigo/lms/templates/discovery/course_card.underscore]: I've gone and update it to be `/theming/asset/images/no_course_image.png`

File: tutorindigo/templates/indigo/lms/templates/course.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <article class="course" id="${course.id}" role="region" aria-label="${course.display_name_with_default}">
     <header class="course-image">
       <div class="cover-image">
-        <img src="${course.course_image_url}" onerror="this.src='/static/indigo/images/no_course_image.png';" alt="${course.display_name_with_default} ${course.display_number_with_default}" />
+        <img src="${course.course_image_url}" onerror='this.src="${static.url('images/no_course_image.png')}";' alt="${course.display_name_with_default} ${course.display_number_with_default}" />
```
Comment (misilot) [inline tutorindigo/templates/indigo/lms/templates/course.html:11]: @hinakhadim should I do the same here?

Comment (hinakhadim) [reply tutorindigo/templates/indigo/lms/templates/course.html:11]: Yes, for consistency. Do the same here.

Comment (misilot) [reply tutorindigo/templates/indigo/lms/templates/course.html]: Done

Comment (hinakhadim) [reply tutorindigo/templates/indigo/lms/templates/course.html]: Thanks

**Pull request merged**